templates/terraform/examples/identity_platform_default_supported_idp_config_basic.tf.erb

```diff
@@ -1,5 +1,6 @@
 resource "google_identity_platform_default_supported_idp_config" "<%= ctx[:primary_resource_id] %>" {
   enabled       = true
-  client_id     = "playgames.google.com"
+  idp_id        = "playgames.google.com"
```

Comment:
Does this mean that anyone who previously configured GCIP will now have to change their config?

Reply:
Yes I don't believe anyone could have successfully configured their IDP config with the client_id and the idp_id as the same value. Generally the `idp_id` would be a generated value, so the API accepts any value, but the correct value would need to be issued by the IDP that is being used for authentication (playgames.google.com in this case).

At least that's my (likely incomplete) understanding of OAuth at this point